modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,59 @@
+= Accessing the console
+
+The {app-security-service} Console is viewed by opening the console URL in a browser. This URL is seen in the list of available services in the Mobile Developer Console (MDC) after the {app-security-service} Service has been provisioned.
+
+= Console overview
+
+The {app-security-service} Console allows you to monitor applications, their respective versions, and disable versions of those applications.
+
+Opening the console presents the applications view. Clicking on an application opens a details view for that application.
+
+Below is a detailed description of each of these views and the information you can expect each to contain.
+
+= Applications view
+
+The applications view lists details for your applications:
+
+* *App Name*: The name of the application.
+* *App ID*: The fully qualified and unique name of the application.
+* *Deployed Versions*: Total number of currently deployed versions.
+* *Current Installs*: Total number of current installed versions of the application.
+* *Launches*: Total number of application launches.
+
+Clicking on an application in the table opens a view with further information and the ability to disable versions of the application.
+
+= Application details view
+
+This view provides statistics and information about each deployed version of an application:
+
+
+* *App Version*: The name of the application.
+* *Current Installs*: Total number of currently installed versions.
+* *Launches*: Total number of launches of this version of the application.
+* *Last launched*: The last time this application version was launched.
+* *Disable on Startup*: Whether this application version is disabled on startup.
+* *Custom Disable Message*: A custom message that can be included to note the reason this application version has been disabled.
```

Comment:
might not be used for this purpose, eg could say to upgrade.
*Custom Disable Message*: A custom message that is displayed when this version of the mobile app is disabled.